### PR TITLE
feat: add finops page

### PR DESCRIPTION
**Pull request by abtx**
- adds FinOps page as per [this spec](https://www.notion.so/infracost/FinOps-landing-page-Jul-23-a9a9249466e549548b8e22d8320f1dce)

Optimization notes:
Removing the parallax effect helps get the desktop site performance up only by 3 points and doesn't affect mobile performance. 

SEO - if there are any SEO keywords TBA to the page, please do let me know.

### CODE REVIEW

File: src/components/FinopsSection/RequestDemoModal.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          </a>{" "}
+          to request a demo.
+        </p>
+        <div className="finops-request-demo__modal-close">
+          <button
+            className="button primary"
+            onClick={() => setModalVisible(false)}
+          >
+            Close
+          </button>
```
Comment (vdmgolub) [inline src/components/FinopsSection/RequestDemoModal.tsx:44]: On a white background, this button looks too intense to me. Can we tone it down a bit? Maybe make a smaller shadow. What do you think?
image

**Review by vdmgolub — APPROVED** [review]

File: src/css/finops.css [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+.finops .finops-wrapper {
+  padding-bottom: 3rem;
+}
+
+.finops-wrapper {
+  gap: 4rem;
+	height: calc(100vh - var(--navbar-height));
+	overflow: hidden;
+	width: 100%;
+	position: relative;
```
[the remark this answers is not in the document]
Comment (abtx) [reply src/css/finops.css:12]: hmm, thanks, let me fix

File: src/components/FinopsSection/RequestDemoModal.tsx (hunk 1 above)
Comment (abtx) [reply src/components/FinopsSection/RequestDemoModal.tsx:44]: sure

File: package.json [hunk 3]
```diff
@@ -20,6 +20,8 @@
     "@docusaurus/theme-classic": "2.2.0",
     "@docusaurus/theme-common": "2.2.0",
     "@mdx-js/react": "^1.6.17",
+    "@react-spring/parallax": "^9.7.3",
```
[the remark this answers is not in the document]
Comment (abtx) [reply package.json:23]: I've noticed, but the alternative is ditching this library and replacing with a new one.

**Pull request merged**